fenicsprecice/adapter_core.py

```diff
@@ -146,19 +156,13 @@ def get_coupling_boundary_vertices(mesh_fenics, coupling_subdomain, fenics_dimen
             vertices_x.append(v.x(0))
             if dimensions == 2:
                 vertices_y.append(v.x(1))
-            elif fenics_dimensions == 2 and dimensions == 3:
-                vertices_y.append(v.x(1))
-                vertices_z.append(0)
```

Comment:
This means that after merging this PR `fenics_dimensions` and `dimensions` are always the same. Correct? Can we get rid of one of the two members then? Note that we should still check that the dimension of the fenics solver (`fenics_dimensions`) matches the dimension provided in the `precice-config.xml` (`dimensions`). I assume this is something we can check once during initialization/configuration of the adapter and then we should only store a single member `dimensions` (if necessary).